nss-day-cohort-60/national-parks-django-api
# Modified SQLite insert queries and models

Modified order of data for several INSERT queries to insert data in correct order.
Changed User relationship from OnetoOne to Foreign key on some models to represent the relationship correctly.


Testing:

1. Delete db.sqlite3
2. delete migrations
3. run python3 manage.py makemigrations in terminal
4. run python3 manage.py migrate in terminal
5. run python3 manage.py loaddata users.json in terminal
6. In VS Code, open national_park_add_data.sql
7. select all contents, run all queries
8. Check db contents for completeness and correctness
